src/Analysers/AttributeAnnotationFactory.php

```diff
@@ -74,6 +74,22 @@ public function build(\Reflector $reflector, Context $context): array
                     }
                 }
             }
+
+            if ($reflector instanceof \ReflectionClass) {
+                foreach ($reflector->getReflectionConstants() as $rc) {
+                    foreach ($rc->getAttributes(Property::class) as $attribute) {
+                        $instance = $attribute->newInstance();
+                        if (Generator::isDefault($instance->property)) {
+                            $instance->property = $rc->getName();
+                        }
+                        if (Generator::isDefault($instance->type)) {
+                            $instance->type = 'const';
```

Comment:
As far as I can see 'const' is not a valid type in OAS?

Reply:
Yes, I am not arguing the const keyword, but the spec does not list `const` as a valid value for `type`: https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.6.1.1, unless I am missing something else here?

Reply:
I would think the `type` should be the type of the `const` value?

Reply:
Sorry I didn't get your point in the first place.
Would you agree that simply removing line 86 is the way to go? This leaves setting the type to the user instead of trying to infer it.

Reply:
I guess that would be good enough to start with.